### ISSUE TITLE
libtbb, Ubuntu repository

### ISSUE THREAD

**Issue by FyodorK** [01org/tbb#61]
Hi, 

Could you please confirm that libtbb 2018 is available from Ubuntu repository?

```
pv@xxxxxxx-x64:/_examples/$ sudo apt-get update
Get:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [107 kB]
Hit:2 http://ru.archive.ubuntu.com/ubuntu xenial InRelease
Hit:3 http://ru.archive.ubuntu.com/ubuntu xenial-updates InRelease                                             
Fetched 107 kB in 0s (161 kB/s)
Reading package lists... Done
```

```
pv@xxxxxxxx-x64:~/_examples/$ apt-cache policy libtbb2 libtbb-dev
libtbb2:
  Installed: (none)
  Candidate: 4.4~20151115-0ubuntu3
  Version table:
     4.4~20151115-0ubuntu3 500
        500 http://ru.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
libtbb-dev:
  Installed: (none)
  Candidate: 4.4~20151115-0ubuntu3
  Version table:
     4.4~20151115-0ubuntu3 500
        500 http://ru.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
```

Thanks in advance.

**Comment by cdluminate** [comment]
tbb 2018 is still not available in Debian or Ubuntu official repository yet.

**Comment by tbbdev** [comment]
TBB team doesn't distribute TBB through Ubuntu official repository, it's done by third-party maintainers.
To get more actual TBB you can use Intel® APT Repository: please see [this instruction](https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo) for details. Now the latest available TBB is 2019.0.
